packages/client/src/declarations/index.ts

```diff
@@ -204,14 +204,6 @@ export interface IVisitedGroupId {
 export interface ITaskHistory {
   operationType?: string
   operatedOn?: string
-  operatorRole?: string
-  operatorName?: Array<GQLHumanName | null>
-  operatorOfficeName?: string
-  operatorOfficeAlias?: Array<string | null>
-  notificationFacilityName?: string
-  notificationFacilityAlias?: Array<string | null>
-  rejectReason?: string
-  rejectComment?: string
```

Comment:
These are altogether unused with the expection of this: https://github.com/opencrvs/opencrvs-core/blob/e55f42307766800266213255a6d011a482cab413/packages/gateway/src/features/search/type-resolvers.ts#L424

I'll figure out if we should keep "notificationFacilityName" or get it some other way.

Reply:
eventLocationId should handle this fine. The old code was this:
```
const facility = getFromBundleById<SavedLocation>(
      bundle,
      body.eventLocationId
    ).resource
    operationHistory.notificationFacilityName = facility?.name || ''
    operationHistory.notificationFacilityAlias = facility?.alias || []
```
which was only added if the composition is doc-type birth/death-notification